.github/workflows/ci.yaml

```diff
@@ -24,6 +25,9 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
         run: just install
+      - name: Install lowest supported Ruff version
+        if: ${{ matrix.ruff == 'lowest' }}
+        run: pip install ruff==$(./get-oldest-supported-version.py ruff)
```

Comment:
I also wouldn't object to just hard-coding this and adding pointer comments between here and the `pyproject.toml` or similar.